cves/CVE-2016-4461.yml

```diff
@@ -212,4 +239,11 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    Overall, this vulnerability stems from failure to consider the security principle of "secure by default".
```

Comment:
You mention that this issue stems from a failure to consider the principle of "secure by default" but you did not mark it in the lessons section for this report. I think Secure by Default is definitely a principle that could be applied here, so perhaps add it into the lessons section.

Reply:
Done. Very good call.